discord.html

```diff
@@ -1,11 +1,37 @@
 <!doctype html>
-<html>
+<html lang='en'>
+
 <head>
-<title>Handshake Discord</title>
-<meta name="description" content="Handshake Discord redirect">
-<meta http-equiv="Refresh" content="0; url='https://discord.gg/Vq3PWF6cJ6'" />
+    <title>Handshake Discord Server</title>
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <meta http-equiv='Content-Type' content='text/html; charset=utf-8' />
+    <meta name="description" content="Decentralized certificate authority and naming" />
+
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="https://handshake.org/discord">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Handshake Discord Server">
+    <meta property="og:description" content="Decentralized certificate authority and naming">
+    <meta property="og:image" content="https://www.handshake.org/images/landing/logo-dark.svg">
+
+    <!-- Twitter Meta Tags -->
+    <meta name="twitter:card" content="summary_large_image">
+    <meta property="twitter:domain" content="handshake.au">
```

Comment:
What's handshake.au ?

Reply:
Oh a typo as I'm so used to using the .au tld.

Reply:
Just fixed it in 92beff3c4254d7019525c2873cb2c684ce4751c1